airflow/providers/microsoft/azure/log/wasb_task_handler.py

```diff
@@ -189,6 +189,7 @@ def wasb_write(self, log: str, remote_log_location: str, append: bool = True) ->
                 log,
                 self.wasb_container,
                 remote_log_location,
+                overwrite=True
```

Comment:
```suggestion
                overwrite=append
```
I think we should use the function parameter instead of hard-coding